spec/linter-eslint-spec.js

```diff
@@ -1,12 +1,17 @@
 'use babel'
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
The default [from airbnb](https://github.com/airbnb/javascript/blob/0157e69a0df8ee8fa568d2c0ed53c087e4910c80/packages/eslint-config-airbnb-base/rules/imports.js#L69) is `"devDependencies: false"` , so it was complaining about `rimraf` not being in `dependencies` in the `package.json`.